flutter/lib/common.dart

```diff
@@ -1844,6 +1853,14 @@ enum UriLinkType {
   rdp,
 }
 
+void handleUriLinkMobile(String uri) {
+  var context = Get.context;
+  var uri_id = uri.split("//").last;
+  if (context != null && uri_id.isNotEmpty){
+    connect(context, uri_id);
+  }
+}
+
```

Comment:
Please use handleUriLink, let mobile and desktop support the same uri format.

Reply:
handleUriLink has multiple cases apart from connection (such as file_transfer, relay, etc). Should I support those as well? or just for connection?

Reply:
For all that are applicable to mobile.